sdk/identity/identity/src/credentials/azureCredential.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { AccessToken } from "./accessToken";
+import { TokenCredential, RequestOptionsBase } from "@azure/core-http";
+import { IdentityClient, IdentityClientOptions } from "../client/identityClient";
+
+export const TokenRefreshBufferMs = 2 * 60 * 1000; // 2 Minutes
```

Comment:
Does this need to be exported?

Reply:
I'm exporting it from this file so that it can be used in `azureCredential.spec.ts` but I didn't include it `index.ts` since it shouldn't be exported to the library's public API.